move2kube/m2k-func/pom.xml

```diff
@@ -63,7 +63,7 @@
     <dependency>
       <groupId>com.github.mwiede</groupId>
       <artifactId>jsch</artifactId>
-      <version>0.2.9</version>
+      <version>0.2.17</version>
```

Comment:
Reading https://github.com/eclipse-jgit/jgit/blob/da37921221075f024b3a4107e12830c1c622e435/org.eclipse.jgit.ssh.jsch/README.md#L4
We should drop this ssh library in favour of the officially and well maintained apache Mina ssh

Reply:
I was referring to the what they say, that `org.eclipse.jgit.ssh.apache` is officially supported, where as the mweide.jsch is by an individual contributor with no guarantees . 
This is high priority.

Reply:
PTAL #254